### PR TITLE
added extra fun commands

### PR DESCRIPTION
**Pull request by marblesx**

### CODE REVIEW

File: miscellaneous/miscellaneous.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+    let RPSScore = [];
+
+
+    /***
+     * Gets message from 8Ball
+     * returns Returns random string.
+     * */
+    function EightBall() {
+        return eightballPhrases[Math.floor(Math.random()*eightballPhrases.length)];
```
Comment (fasteddeh) [inline miscellaneous/miscellaneous.js]: could be reading this wrong, but this makes it so phrases can be easily added without breaking code, correct? cause it grabs the length of the phrases list then makes that the number that is the ceiling for the random function?

Comment (marblesx) [reply miscellaneous/miscellaneous.js]: correct, ideally I'd want this in a DB

File: miscellaneous/miscellaneous.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            else
+            {
+                user.w++;
+                message = 'You picked paper, cpu picked scissors. You won. <@'+ userID +'>\n' +
+                    ' wins ' + user.w +
+                    '\nloses '+ user.l +
+                    '\nties '+ user.t
+            }
+        }
+        let newUser = true;
```
Comment (fasteddeh) [inline miscellaneous/miscellaneous.js]: starting from this point I get lost on what's going on

Comment (fasteddeh) [reply miscellaneous/miscellaneous.js]: I imagine that it's recording the record of the player and figuring out if it's a new player or not.

**Pull request merged**